### PR TITLE
Looking for unused fields and deleting them from source

### PR DESCRIPTION
**Pull request by coveomarya**
Let me know if there are any issues

### CODE REVIEW

File: migrate_fields_config.py [hunk 1]
```diff
@@ -218,4 +230,9 @@ def v2_get_updated_fields(field_differences):
         v2_client.update_fields(v2_fields_updated)
     else:
         print('No fields to update.')
-    print('Migration completed.')
\ No newline at end of file
+    print('Migration completed.')    
+    #Deleting unused fields
+    unusedfields = get_unused_fields(v2_client.get_fields_with_mappings())
```
Comment (malaporte) [inline migrate_fields_config.py]: maybe this should be done only when a specific option is passed?

**Review by malaporte — APPROVED** [review]

File: migrate_fields_config.py (hunk 1 above)
Comment (coveomarya) [reply migrate_fields_config.py]: Added a --delete_fields option, and also check if the item has sytem == false before deleting

**Review by malaporte — APPROVED** [review]

**Comment by mireaulf** [comment]
@coveomarya Can you also update the readme in this PR?

**Comment by coveomarya** [comment]
@mireaulf I already updated it?

**Pull request merged**